x/rollapp/keeper/rollapp_suite_test.go

```diff
@@ -70,6 +75,16 @@ func (suite *RollappTestSuite) SetupTest(deployerWhitelist ...types.DeployerPara
 	suite.queryClient = queryClient
 }
 
+func (s *RollappTestSuite) keeper() *keeper.Keeper {
+	return s.App.RollappKeeper
+}
+
+func (s *RollappTestSuite) nextBlock(dt time.Duration) {
```

Comment:
this added duration seems a bit shoehorned in, IMO

Reply:
fixed